docs/content/about/tutorial.md

```diff
@@ -10,63 +10,69 @@ This tutorial covers the basics to get started and to use Nikita. It is organize
 - [What is Nikita?](#what-is-nikita)
 - [Installation instructions](#installation-instructions)
 - [Core concepts](#core-concepts)
-- [Real life example](#real-life-example)
+- [Real-life example](#real-life-example)
 
 Feel free to skip the second section if you are familiar with Node.js and its ecosystem.
 
 For detailed information, navigate the documentation or submit an issue if you don't find the answers to your questions. Also, if you are looking for examples, the source code is well documented and its test coverage quite complete. We highly encourage you to navigate the tests. Tests are self contained and very easy to understand. They also provide you the guaranty of reading a working code.
 
 ## What is Nikita?
 
-Nikita is build as a library, which provides simple functions on a host (server, desktop, machine, vm...) either locally or remotely over ssh.
+Nikita is build as a library, which provides simple functions on a host (server, desktop, machine, vm...) either locally or remotely over SSH.
 
 ### Technologies
 
-Nikita is written in JavaScript and executed by NodeJs. It is delivered as a Node.js package and is available on NPM.
+Nikita is written in JavaScript and executed by NodeJs. It is delivered as a Node.js package and is available on [NPM](https://www.npmjs.com/package/nikita).
 
 ### Use cases
 
-It can serve  multiple purposes. For example, it can be used in a website with a Node.js backend, where you want to execute actions (writing files, copy, move, executing custom scripts...) or you can use it to automate and orchestrate  components' deployments (installations, functional tests, lifecycle management...).
+It can serve multiple purposes. For example, it can be used in a website with a Node.js backend, where you want to execute actions (writing files, copy, move, executing custom scripts...) or you can use it to automate and orchestrate components' deployments (installations, functional tests, lifecycle management...).
 
-Take a view at [Ryba](https://github.com/ryba-io/ryba) which contains playbooks to setup and manage Big Data system.
+Take a view at [Ryba](https://github.com/ryba-io/ryba) which contains playbooks to setup and manage Big Data systems.
+
+### Supported platforms
+
+Nikita targets Unix-like system including Linux and macOS. Windows is not supported as a targeting node where to execute actions. It is however known to work as a Nikita host. This mean you can run Nikita from a Windows host as long as you are targeting Linux nodes over SSH.
+
+Throughout this tutorial, it is assumed you work on Linux or macOS. To be able to run the same code examples without modifications on Windows, the easiest way would be to work inside a [Node.js Docker container](https://hub.docker.com/_/node). Before that, you have to install Docker Desktop following [this guide](https://docs.docker.com/docker-for-windows/install/) and start a container with the command `docker run -it node bash`.
```

Comment:
Throughout this tutorial, it is assumed you work on Linux or macOS. To be able to run the same code examples without modifications on Windows, you can download a Virtual Machine or use Docker.

Reply:
At the end of the tutorial, you will learn how to use Nikita over SSH. This way, your windows host is used to create your workflow and you can target a remote hosts like a server, a virtual machine or a container.